`test_refreshable_mat_view/test.py::test_real_wait_refresh[to_clause1-False-True]` is flaky

Here: https://s3.amazonaws.com/clickhouse-test-reports/72823/4ee4aa08a6193724e7d50a58abb6f343876946af/integration_tests__aarch64__[6_6].html

```
=================================== FAILURES ===================================
________________ test_real_wait_refresh[to_clause1-False-True] _________________
[gw0] linux -- Python 3.10.12 /usr/bin/python3

fn_setup_tables = None, append = True, empty = False
to_clause = ('Engine MergeTree ORDER BY tuple()', None, 'test_rmv')

    @pytest.mark.parametrize(
        "append",
        [True, False],
    )
    @pytest.mark.parametrize(
        "empty",
        [True, False],
    )
    @pytest.mark.parametrize(
        "to_clause",
        [
            (None, "tgt1", "tgt1"),
            ("Engine MergeTree ORDER BY tuple()", None, "test_rmv"),
        ],
    )
    def test_real_wait_refresh(
        fn_setup_tables,
        append,
        empty,
        to_clause,
    ):
        if node.is_built_with_sanitizer():
            pytest.skip("Disabled for sanitizers (too slow)")
    
        table_clause, to_clause_, tgt = to_clause
    
        create_sql = CREATE_RMV.render(
            table_name="test_rmv",
            refresh_interval="EVERY 10 SECOND",
            to_clause=to_clause_,
            table_clause=table_clause,
            select_query="SELECT now() as a, b FROM src1",
            with_append=append,
            empty=empty,
        )
        node.query(create_sql)
        rmv = get_rmv_info(node, "test_rmv")
    
        expected_rows = 0
        if empty:
            expect_rows(expected_rows, table=tgt)
        else:
            expected_rows += 2
            expect_rows(expected_rows, table=tgt)
    
        rmv2 = get_rmv_info(
            node,
            "test_rmv",
            condition=lambda x: x["last_refresh_time"] == rmv["next_refresh_time"],
            # wait for refresh a little bit more than 10 seconds
            max_attempts=30,
            delay=0.5,
        )
    
        if append:
            expected_rows += 2
            expect_rows(expected_rows, table=tgt)
        else:
            expect_rows(2, table=tgt)
    
        assert rmv2["exception"] is None
        assert rmv2["status"] in ["Scheduled", "Running"]
>       assert rmv2["last_success_time"] == rmv["next_refresh_time"]
E       assert datetime.datetime(2024, 12, 17, 15, 56, 15) == datetime.datetime(2024, 12, 17, 15, 56, 20)

test_refreshable_mat_view/test.py:392: AssertionError
```

It is interesting that the PR which fails already included [another PR](https://github.com/ClickHouse/ClickHouse/pull/72823) which should (theoretically) address the issue. Seems like the fix does not reliably work. 

@fm4v

--- a/tests/integration/test_refreshable_mat_view/test.py
+++ b/tests/integration/test_refreshable_mat_view/test.py
@@ -389,7 +389,6 @@ def test_real_wait_refresh(
 
     assert rmv2["exception"] is None
     assert rmv2["status"] in ["Scheduled", "Running"]
-    assert rmv2["last_success_time"] == rmv["next_refresh_time"]
     assert rmv2["last_refresh_time"] == rmv["next_refresh_time"]
     assert rmv2["retry"] == 0
     assert rmv2["read_rows"] == 2

